java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -98,6 +99,17 @@ open class DevToolsService : Service() {
         devToolsServer.send(service.storageKeys ?: "")
       }
 
+      devToolsServer.addOnMessageCallback { message ->
```

Comment:
Should you ever remove this callback? What will happen if user toggles DevTools off? If we don't yet handle it gracefully we should probably file a bug. And we should probably have an umbrella bug will all the potential DevTools related fixes we should someday do, I don't think we have set it up yet.

You can also ping me directly and we can set that up together.

Reply:
+1

Reply:
Parent bug is b/154554631 added a sub-bug to track this: b/170155112